qiskit/schemas/qobj_schema.json

```diff
@@ -153,6 +154,30 @@
                     ],
                     "title": "Frame change pulse"
                 },
+                {
+                    "properties": {
+                        "name": {
+                            "enum": [
+                                "delay"
+                            ]
+                        },
+                        "duration": {
```

Comment:
Why is the definition of duration so complex? Could it be something similar to how t0 is defined below? 
```json
"t0": {
           "minimum": 0,
            "type": "integer"
                }
```

Reply:
ok. I've updated this to
```
"duration": { type: "integer" }
```